CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.11.16] - 2025-04-04
```

Comment:
Please remove this line, it will be added automatically when publishing the release